deepchem/models/losses.py

```diff
@@ -205,6 +205,162 @@ def loss(output, labels):
     return loss
 
 
+class VAE_ELBO(Loss):
```

Comment:
Could you add all these classes to the autodoc on `docs/models.rst` in the Losses section? You just need to add `autoclass` statements as for the other classes. This should hopefully be clear how to do from looking at the link here:

https://github.com/deepchem/deepchem/blob/master/docs/models.rst#L169

Reply:
Finished.